src/rust/ide/view/graph-editor/src/component/edge.rs

```diff
@@ -1207,19 +1205,22 @@ impl Edge {
 
             color_update <- all(input.set_color,input.set_dimmed);
 
-            eval color_update ([edge_color,style]((color,should_dim)) {
-                let target_color = if *should_dim {
-                   style.get_color_dim(*color)
-                } else {
-                  color.into()
-                };
-                edge_color.set_target(target_color);
+            // TODO: dimming commented for now, as it was using bad colors. Should be mixed with background instead.
```

Comment:
But that makes one node's button without any visual effect, doesn't it? @sylwiabr is that ok from UX perspective?

Reply:
So I will not see if the node is active or am I pressing visualization button? If so then no, it is not ok.

Reply:
fixed